### ISSUE TITLE
Remove stale branches?

### ISSUE THREAD

**Issue by weaverba137** [desihub/desiconda#39]
The `basemap` and `collabsu` branches have already been merged, `18.5` is presumably quite out-of-date, and `noconda` I'm guessing is an experiment from a few years ago.

**Comment by sbailey** [comment]
I deleted basemap, collabsu, and 18.5 branches.

@tskisner the `noconda` branch appears to have some potentially useful documentation about virtualenvs and config files for installing dependencies with package managers instead of conda.  Do you think any of that should be preserved, or at this point is it so old that the act of testing/verifying the instructions would be as much work as re-creating them in the future if needed?  Please take a look and comment.  thanks.

**Comment by tskisner** [comment]
I guess the point of that branch was that all the conda packages installed by desiconda are also available as pip packages.  The real "value add" of desiconda is building the compiled dependencies in a consistent way.  Perhaps someday specex will turn into a python package and we can convert all of desiconda into a small list of packages to install with conda or pip.

I've forked this repo so that I have a copy of the branch for my own reference.  I also rebased the branch in this upstream repo.  Feel free to delete it if you like.
